snap/squashfs/export_test.go

```diff
@@ -88,3 +88,11 @@ func MockIsRootWritableOverlay(new func() (string, error)) (restore func()) {
 		isRootWritableOverlay = old
 	}
 }
+
+func MockTruncateSnapToMinSize(newTruncate func(string, int64) error) (restore func()) {
```

Comment:
Is this the right strategy for mocking here? Some of the other tests mock mksquashfs, so we don't have a file to actually truncate.

Reply:
Generally, we use mocking if we want to simulate the effects of calls to external functions, syscalls, etc. If we mock internal functions, then those functions do not actually get tested whenever they are mocked (as you know). From what I can see, the tests use this function mocking to make the function a no-op. Is there a particular reason not to let the truncation code run in those other tests?